### PR TITLE
Add support for `sharp-duotone-solid` icons

### PR DESCRIPTION
**Pull request by JackWH**
This PR adds support for the [new `sharp-duotone-solid` variation](https://blog.fontawesome.com/introducing-sharp-duotone/), released with FontAwesome 6.6.0.

Thanks!

### CODE REVIEW

**Comment by JackWH** [comment]
Worth noting, I set the prefix in the config file to `fasds`. This seems to differ from the convention used by the other prefixes.

I don't know how the other sharp set prefixes were determined, but `fasds` is the only way I could get these to load with Blade Icons. In my project I've had to override the config to change the prefixes for all the other Pro sets too (e.g. `sharp-solid` should be `fass`, `sharp-thin` should be `fast`, etc.), otherwise the Blade Icons package refused to load them.

Not sure if that's specific to my setup or just incorrect historically, but FYI @owenvoke in case that needs changing.

File: config/blade-fontawesome.php [hunk 1]
```diff
@@ -149,6 +149,21 @@
 
     ],
 
+    'sharp-duotone-solid' => [
+    
+        'prefix' => 'fasds',
```
Comment (owenvoke) [inline config/blade-fontawesome.php:154]: The prefixes seem to have never worked for the ones with dashes in, I've merged #77 and opened #85 to see if I can get around not having a prefix separator with a colon.

Would you be able to test if this works for you? 🤔 Basically by changing this from `fasds` to `fas:ds` (and then using it via Blade).

I've tested it locally, but I don't have Pro anymore, so can't really test easily.

**Review by owenvoke — COMMENTED** [review]
Sorry for the delayed review on this, and thanks for the contribution. I've left a small query below.

File: config/blade-fontawesome.php (hunk 1 above)
Comment (ToonDG) [reply config/blade-fontawesome.php:154]: I tried adding sharp-duotone-solid using `fas:ds`, and it worked.

**Review by owenvoke — APPROVED** [review]

**Comment by owenvoke** [comment]
Thank you, @JackWH! Appreciate this. I'll fix the Pint changes separately, and then release.

**Pull request merged**